docs/volunteering/srawn.md

```diff
@@ -0,0 +1,79 @@
+# SR(A)WN
+
+The Student Robotics (Almost) Weekly Newsletter (SR(A)WN) is a semi-regular newsletter written by volunteers for volunteers.
+
+## Availablility
+
+SR(A)WN can be consumed electronically from several places:
+
+- Join the `#srawn` channel on [Slack](https://app.slack.com/client/T0EEPF1LH/C01GBT8NMSN) _recommended_
```

Comment:
Suggest linking to the channel from the channel name rather than from the word "Slack".
This would allow linking to either our signup form or our docs on Slack from the word Slack if we wanted to.
```suggestion
- Join the [`#srawn` channel](https://app.slack.com/client/T0EEPF1LH/C01GBT8NMSN) on Slack _recommended_
```
It may also make sense to do this for the mailing list too?